cmd/neofs-node/cache.go

```diff
@@ -319,3 +319,32 @@ func (s *ttlContainerLister) InvalidateContainerListByCID(id *cid.ID) {
 		}
 	}
 }
+
+type cachedIRFetcher ttlNetCache
+
+func newCachedIRFetcher(f interface{ InnerRingKeys() ([][]byte, error) }) *cachedIRFetcher {
+	const (
+		irFetcherCacheSize = 1 // we intend to store only one value
```

Comment:
Do we plan to increase this? If not, maybe implement a cache directly, instead of reusing `ttlNetCache` and all `lru` machinery useless for this case?

Reply:
We don't plan to increase it.  

However I think it is better to re-use existing syncing mechanism instead of creating one more cache with mutex. What do you think?